addons/payment_stripe/models/payment_acquirer.py

```diff
@@ -0,0 +1,131 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+import hmac
+import logging
+import requests
+
+from datetime import datetime
+from hashlib import sha256
+from requests.exceptions import HTTPError
+from werkzeug import urls
+
+from odoo import api, fields, models, _
+from odoo.http import request
+from odoo.tools import consteq
+from odoo.exceptions import ValidationError
+
+_logger = logging.getLogger(__name__)
+
+STRIPE_SIGNATURE_AGE_TOLERANCE = 600  # in seconds
+
+
+class PaymentAcquirer(models.Model):
+    _inherit = 'payment.acquirer'
+
+    provider = fields.Selection(selection_add=[
+        ('stripe', 'Stripe')
+    ], ondelete={'stripe': 'set default'})
+    stripe_secret_key = fields.Char(required_if_provider='stripe', groups='base.group_system')
+    stripe_publishable_key = fields.Char(required_if_provider='stripe', groups='base.group_system')
```

Comment:
Please add a comment to explain that there is no problem if that key is exposed (I was confused at first when I saw that a route made the information public ^^).
```suggestion
    stripe_publishable_key = fields.Char(
        required_if_provider='stripe', groups='base.group_system',
        help="The publishable key of your Stripe account. Can be safely exposed.")
```